modules/lightning_features/lightning_layout/modules/lightning_inline_block/src/LightningInlineBlockServiceProvider.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Drupal\lightning_inline_block;
+
+use Drupal\Core\DependencyInjection\ContainerBuilder;
+use Drupal\Core\DependencyInjection\ServiceProviderBase;
+
+class LightningInlineBlockServiceProvider extends ServiceProviderBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function register(ContainerBuilder $container) {
+    parent::register($container);
+
+    if ($container->hasDefinition('lightning.quickedit_access')) {
+      $container
+        ->getDefinition('lightning.quickedit_access')
+        ->setClass(QuickEditAccess::class);
```

Comment:
QuickEditAccess is implemented by Lightning Workflow, which we cannot depend on -- thus the need for a service modifier. Ultimately I think this should be merged into core, but I will have to talk to @wimleers about that.